packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -132,10 +133,10 @@ module.exports = {
     ],
 
     sidebars: {
-      codePages: require('./nav/codePages'),
-      reference: require('./nav/reference'),
-      guides: require('./nav/guides'),
-      concepts: require('./nav/concepts')
+      codePages: redesign ? require('./nav/newNav/languagesSdk') : require('./nav/codePages'),
```

Comment:
Can we keep it to `redesign` here instead of `newNav`? Will be much easier to find and replace in a couple of weeks.